.github/workflows/dependabot-assignments.yml

```
@@ -12,6 +12,13 @@ jobs:
     if: ${{ github.actor == 'dependabot[bot]' }}
     steps:
       - name: Assign team to PR
-        run: gh pr edit "$PR_URL" --add-reviewer "cmu-delphi/code-reviewers"
-        env:
-          PR_URL: ${{github.event.pull_request.html_url}}
\ No newline at end of file
+        # This is a workaround a limitation in the GitHub CLI.
+        # Can probably simplify after this issue is closed:
+        # https://github.com/cli/cli/issues/4844
+        run: | 
+          curl -X POST \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}"\
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            https://api.github.com/repos${{ github.owner }}/${{ github.repository }}/pulls/${{github.event.pull_request.number}}/requested_reviewers \
```

I suspect the linked workaround wasn't copy-pasted from a working workflow, but this should do it if I'm reading the docs correctly:

```suggestion
            https://api.github.com/repos/${{ github.repository }}/pulls/${{github.event.pull_request.number}}/requested_reviewers \
```

From [context docs](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context):

> github.repository - string -  The owner and repository name. For example, `octocat/Hello-World`.

& from [REST docs for requesting reviewers](https://docs.github.com/en/rest/pulls/review-requests?apiVersion=2022-11-28):

> `/repos/{owner}/{repo}/pulls/{pull_number}/requested_reviewers`